datajoint/connection.py

```diff
@@ -17,7 +17,7 @@
 logger = logging.getLogger(__name__)
 
 
-def conn(host=None, user=None, password=None, init_fun=None, reset=False):
+def conn(host=None, user=None, password=None, init_fun=None, reset=False, use_ssl=None):
```

Comment:
change SSL to TLS throughout. SSL has been deprecated. It does not matter that others are not caught up yet.

Reply:
Addressed